source/help/getting-started/organizing-conversations.rst

```diff
@@ -154,6 +154,8 @@ When configured by the System Admin, users can enable the features in **Account
 
 **Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
 
+**Move channels and categories**: Specify a destination for selected channels using the **Move to** under the **More options** icon. This feature is available today for Cloud deployments and will be available for self-hosted deployments in Mattermost Server v5.32 and later. 
```

Comment:
@cwarnermm I think the way the bullets are ordered here lessens the impact of drag and drop which is the primary way for people to move channels. I think of "move to" as a secondary way to move channels, hence why it may be better as a sub point under drag and drop. Also move to existed before v5.32, it's just that you can now multi-select > move to that's the new feature, so probably not even worth complicating the docs for that. 

Propose:

**Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.

**Drag and drop channels and categories**: Drag selected channels between or within categories, or drag to reorder entire categories to prioritize important conversations. You can also specify a destination for selected channels using the **Move to** under the **More options** icon.

Reply:
Thanks, @esethna. This section underwent a number of individual changes recently, and it's good to step back and look at the section as a whole.